### ISSUE TITLE
Drag and drop block to end of group shows scroll arrows under the insert block icon

### ISSUE THREAD

**Issue by bobbingwide** [WordPress/gutenberg#24248]
**Describe the bug**
After dragging a block into a group block, to be position after all the other blocks that are already there, the Add block icon was displayed in the outer group block **with up and down scroll bar arrows**... as if there was an overflow problem.

**To reproduce**
1. Create a paragraph inside a group inside a group
2. Add another block after the group. e.g. a spacer
3. Drag the new block after the nested group
4. See the Add block icon appear inside the group block, with up/down arrows behind it.

**Expected behavior**
I didn't expect to see the up/down arrows.

**Screenshots**
![image](https://user-images.githubusercontent.com/2474435/88689818-defce480-d0f2-11ea-97a0-7e4c94d5b0fd.png)

**Editor version (please complete the following information):**
Problems noted with 
5.5-beta3 and not Gutenberg
5.4.2 and Gutenberg 8.6.0

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] Version 84.0.4147.89 (Official Build) (64-bit)

**Additional context**
-

**Comment by talldan** [comment]
Hey @bobbingwide, I have a feeling this issue might be theme dependent. I tested TwentyTwenty and didn't spot this problem. Which theme are you using for the test?

**Comment by bobbingwide** [comment]
@talldan. I'll just check
Trouble is, everytime I do something I come across another problem in the way. 
e.g. In 5.5-RC1, when I checked my theme versions to confirm I was using the latest version of Twenty Twenty -  1.4  - I saw 8 Notices 
```
Notice: Undefined variable: compatible_php in C:\apache\htdocs\wp55\wp-admin\includes\update.php on line 649
```

**Comment by bobbingwide** [comment]
Here's a screenshot with WordPress 5.5-RC1 and not Gutenberg using TwentyTwenty 1.4

In order to get the problem to appear I needed to set background colours on both the outer and inner group.
I've updated the Steps to reproduce in the issue's original comment. 

![image](https://user-images.githubusercontent.com/2474435/89025408-b48f6f00-d31e-11ea-9ee8-3a490b97ce4e.png)

**Comment by bobbingwide** [comment]
> Notice: Undefined variable: compatible_php in C:\apache\htdocs\wp55\wp-admin\includes\update.php on line 649

Looks like this problem has been fixed in https://core.trac.wordpress.org/changeset/48690

**Comment by tellthemachines** [comment]
Hey @bobbingwide , this appears to have been fixed as of WP 5.6. Closing the issue but feel free to reopen if you can still reproduce it.
